app/tests/test_download.py

```diff
@@ -0,0 +1,124 @@
+import boto3
+from moto import mock_s3
+
+from app.tests.conftest import mock_standard_user, mock_superuser
+from app.tests.factories import (
+    ConsignmentFactory,
+    FileFactory,
+    FileMetadataFactory,
+)
+
+BUCKET = "test-download"
+CONSIGNMENT_REF = "TDR-123-TEST"
+FILE_PATH = "data/content/folder_a/test_file.txt"
+
+
+def test_invalid_id_raises_404(client):
```

Comment:
Can we put this below the setup functions, alongside the other tests, for consistency / easiert to read this file.